src/Main.java

```diff
@@ -1,5 +1,346 @@
+            System.out.println("Please enter any integer");
+            numberForTheLoop1b = scanner.nextInt();
+        } while (numberForTheLoop1b != -1000);
+        {
+            System.out.println("The exit number " + numberForTheLoop1b + " has been detected, you are out of the loop!");
+        }
+
+        //Task 1c
+        System.out.println("You are on task 1c \n");
+
+        int totalValueOfTheSum = 0;
+        int numberToAddToTheValue;
+        boolean runArray1c = true;
+
+        while (runArray1c) {
+
+            int previousValueOfTheSum = totalValueOfTheSum;
+            numberToAddToTheValue = enterInteger();
+            if (numberToAddToTheValue == -1000) break;
+            totalValueOfTheSum = addingIntegers(previousValueOfTheSum, numberToAddToTheValue);
+            System.out.println("The total of " + previousValueOfTheSum + " and " + numberToAddToTheValue + " is " + totalValueOfTheSum);
+        }
+
+        //Task 1d
+        System.out.println("You are on task 1d \n");
+
+        int totalValue = 0;
+        int numberForOperation;
+        String operationToPerform;
+        boolean runArray1d = true;
+
+        while (runArray1d) {
+
+            int previousValueOfTheSum = totalValue;
+            operationToPerform = enterOperation();
+
+            if (operationToPerform.equals("="))
+                break; //Adding this as I have offered this operation in function I used for next tasks
+
+            numberForOperation = enterInteger();
+
+            if (operationToPerform.equals("+")) {
+                totalValue = addingIntegers(previousValueOfTheSum, numberForOperation);
+            } else if (operationToPerform.equals("-")) {
+                totalValue = subtractingIntegers(previousValueOfTheSum, numberForOperation);
+            } else if (operationToPerform.equals("*")) {
+                totalValue = multiplyingIntegers(previousValueOfTheSum, numberForOperation);
+            } else if (operationToPerform.equals("/")) {
+                totalValue = divisioningIntegers(previousValueOfTheSum, numberForOperation);
+            }
+            System.out.println("The total of " + previousValueOfTheSum + " and " + numberForOperation + " is " + totalValue);
+        }
+
+        //Task 1e
+        System.out.println("You are on task 1e \n");
+
+        int totalValuePlus = 0;
+        int numberForOperationPlus;
+        String operationToPerformPlus;
+        boolean runArray1e = true;
+
+        while (runArray1e) {
+            int previousValueOfTheSum = totalValuePlus;
+            operationToPerformPlus = enterOperation();
+
+            if (operationToPerformPlus.equals("="))
+                break; //Adding this as I have offered this operation in function I used for next tasks
+
+            numberForOperationPlus = enterInteger();
+
+            if (operationToPerformPlus.equals("+")) {
+                totalValuePlus = addingIntegers(previousValueOfTheSum, numberForOperationPlus);
```

Comment:
why is this print out not in the function it belongs to....it looks like a part of some logic

Reply:
Good point. I didn't add it initially as I have defined the function in the first task, before I needed to printout the actual name of the function result (in the previous task the printout should be "Total of X and Y is Z"). I have corrected it and now I have two of the same tasks, if that is ok.

Reply:
Sve mi je jasno now.